app/services/bulk_mailers/school_reminder_comms.rb

```diff
@@ -73,12 +73,13 @@ def contact_sits_that_need_to_appoint_an_ab_for_unregistered_ects
     def contact_sits_that_need_to_assign_mentors
       query = Schools::WithEctsWithNoMentorQuery.call(cohort:, school_type_codes: SCHOOL_TYPES_TO_INCLUDE)
 
+      query = query.joins(:induction_coordinator_profiles)
+
       return query.count if dry_run
 
       email_count = 0
 
       query
-        .joins(:induction_coordinator_profiles)
         .includes(:induction_coordinator_profiles)
```

Comment:
The query has  both, joins and includes:
 `.joins(:induction_coordinator_profiles).includes(:induction_coordinator_profiles)` no?

Reply:
Yes, I'm using `.includes` to eager load not to left outer join but will switch to `.eager_load` to make the intention clear. Old habits.